cfpb/cfgov-refresh
# Adding Email popup back to cfgov-refresh

Adding Email popup back to cfgov-refresh

## Testing

1. Enable the flag for the appropriate route ( http://localhost:8000/owning-a-home/ )
2. Visit `http://localhost:8000/owning-a-home/`.
3. Scroll to the bottom of the page and verify that the popup appears.
4. Enter in a incorrect email address in the popup and verify the validation works.
5. Enter in a correct email address in the popup and verify the email is submitted correctly.

## Screenshots


## Notes

- I'm certain there are better ways to get emails on file and encourage topic specific engagement. 


## Checklist
